butane_core/src/migrations/fsmigrations.rs

```diff
@@ -20,13 +20,17 @@ const TYPES_FILENAME: &str = "types.json";
 struct MigrationInfo {
     /// The migration this one is based on, or None if this is the
     /// first migration in the chain
+    #[serde(default, skip_serializing_if = "Option::is_none")]
     from_name: Option<String>,
+    #[serde(default, skip_serializing_if = "BTreeMap::is_empty")]
+    existing_schema: BTreeMap<String, String>,
```

Comment:
im not especially happy with this name.

Reply:
Perhaps `table_bases`

Reply:
I was also thinking a name including "inherit" might work.